pkg/resources/reconciler.go

```diff
@@ -229,6 +235,11 @@ func (r *Reconciler) ReconcileSubscription(ctx context.Context, target marketpla
 		}
 	}
 
+	// If all install plans failed - operator is not installed / upgraded
+	if allIpsFailed {
```

Comment:
is it possible we introduce a timing issue here? Could we see all IPs markes as failed before OLM creates a new one?

Reply:
cc @philbrookes

Reply:
this will not abort the install though, if OLM creates another installplan after this returns, it will be picked up on the next reconcile - or have I misread this @KevFan

Reply:
@philbrookes Yes, correct, installation will continue if another install plan is created and does not fail. Though I'm no longer sure will this resolve the issue as I had an install where only one install plan was created even when it failed :thinking: Possibly more of a timing issue